### ISSUE TITLE
Helm output of tyk pro needs to be updated

### ISSUE THREAD

**Issue by sedkis** [TykTechnologies/tyk-helm-chart#49]
This is the current output:

```
$ helm install tyk-pro -f ./values.yaml ./tyk-pro

1. Bootstrap the dashboard so we can get a username and password to login, this also generates access tokens for the controller to use
export NODE_PORT=$(kubectl get --namespace tyk-ingress -o jsonpath="{.spec.ports[0].nodePort}" services dashboard-svc-tyk-pro)
export NODE_IP=$(kubectl get nodes --selector=kubernetes.io/role!=master -o jsonpath='{.items[0].status.addresses[?(@.type=="ExternalIP")].address}')

If you're using minikube, run this instead:
export NODE_IP=$(minikube ip)

export DASH_URL="$NODE_IP:$NODE_PORT"
export DASH_HTTPS=""

1a. You may need to open up that port in your firewall so that we can access the dashboard. Example in GCloud

gcloud compute firewall-rules create dashboard --allow tcp:$NODE_PORT

1b. Bootstrap the dashboard

./tyk-pro/scripts/bootstrap_k8s.sh $DASH_URL 12345 tyk-ingress $DASH_HTTPS

At this point, Tyk Pro is fully installed and should be accessible, proceed in case you want to install Tyk ingress controller.
```

Note that all the bootstrap steps have been replaced by the bootstrap UI.  The user simply needs to navigate to the Dashboard now.

**Note that step 1A might still be necessary**

**Comment by excieve** [comment]
This is more nuanced. UI is not great for automation and in some setups it might be blocked, e.g. if you're using `tyk-k8s` to manage dashboard ingress and your worker node network is not publicly exposed, which is common on production.

In addition to this, there is a bug in current stable `tyk-analytics` that prevents the bootstrap UI to work behind a load balancer (again, something you'd typically have on production). This will be fixed in 1.10.

So I'd suggest to start mentioning bootstrap UI once 1.10 is out, but just as one of the options.

**Comment by sedkis** [comment]
Ok - thx
